### PR TITLE
update node-apis.md

### PR DESCRIPTION
**Pull request by spathak98**

### CODE REVIEW

File: docs/docs/node-apis.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 ---
 title: Gatsby Node APIs
-description: Documentation on Node APIs used in Gatsby build process for common uses like creating pages
+description: पृष्ठे तयार करण्यासारख्या सामान्य वापरासाठी Gatsby बिल्ड प्रक्रियेत वापरल्या जाणार्‍या नोड एपीआयवरील दस्तऐवजीकरण
```
Comment (spielers) [inline docs/docs/node-apis.md]: दस्तऐवजीकरण => it should be documentation as it is

File: docs/docs/node-apis.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 ---
 
-Gatsby gives plugins and site builders many APIs for controlling your site's data in the GraphQL data layer.
+ग्राफ्स डेटा लेयरमध्ये आपल्या साइटचा डेटा नियंत्रित करण्यासाठी Gatsby प्लगइन आणि साइट बिल्डर्सना बरेच एपीआय देते.
 
-## Async plugins
+## एसिंक प्लगइन
 
-If your plugin performs async operations (disk I/O, database access, calling remote APIs, etc.) you must either return a promise or use the callback passed to the 3rd argument. Gatsby needs to know when plugins are finished as some APIs, to work correctly, require previous APIs to be complete first. See [Debugging Async Lifecycles](/docs/debugging-async-lifecycles/) for more info.
+जर आपले प्लगइन async ऑपरेशन्स करीत असेल (डिस्क I / O, डेटाबेस प्रवेश, रिमोट एपीआय कॉल करणे, इ.) आपण एकतर वचन दिले पाहिजे किंवा 3 रा वितर्कात पास केलेला कॉलबॅक वापरणे आवश्यक आहे. काही एपीआय म्हणून प्लगइन समाप्त झाल्यावर गॅटस्बीला हे माहित असणे आवश्यक आहे, योग्यरित्या कार्य करण्यासाठी, आधीचे API पूर्ण करणे आवश्यक आहे. अधिक माहितीसाठी [डीबगिंग एसिंक लाइफसायकल](/docs/debugging-async-lifecycles/)पहा.
```
Comment (spielers) [inline docs/docs/node-apis.md]: please retranslate this

**Review by spielers — CHANGES_REQUESTED** [review]
hey, @spathak98  please do the requested changes

File: docs/docs/node-apis.md (hunk 1 above)
Comment (visshaljagtap) [reply docs/docs/node-apis.md]: also dont translate pages to पृष्ठे, keep as is

**Review by visshaljagtap — CHANGES_REQUESTED** [review]

**Comment by spathak98** [comment]
Hey @spielers I have changed the required changes, please look into it.

**Comment by visshaljagtap** [comment]
LGTM!

**Review by visshaljagtap — APPROVED** [review]

**Review by spielers — APPROVED** [review]
Cheers !!!

**Pull request merged**